### PR TITLE
chore(tests): removed the need for up/down functions in tests

### PR DESCRIPTION
**Pull request by jdalsem**

### CODE REVIEW

File: engine/tests/classes/Elgg/BaseTestCase.php [hunk 1]
```diff
@@ -189,13 +189,15 @@ protected function tearDown(): void {
 	 * Called after setUp() method and can be used by test cases to setup their test logic
 	 * @return mixed
 	 */
-	abstract function up();
+	protected function up() {
```
Comment (jeabakker) [inline engine/tests/classes/Elgg/BaseTestCase.php:192]: better make this `public`

File: engine/tests/classes/Elgg/BaseTestCase.php [hunk 2]
```diff
@@ -189,13 +189,15 @@ protected function tearDown(): void {
...
+	}
 
 	/**
 	 * Called before tearDown() method and can be used by test cases to clear their test logic
 	 * @return mixed
 	 */
-	abstract function down();
+	protected function down() {
```
Comment (jeabakker) [inline engine/tests/classes/Elgg/BaseTestCase.php:199]: better make this `public`

**Review by jeabakker — CHANGES_REQUESTED** [review]
Please add some documentation about usefull helper function in tests

**Comment by jdalsem** [comment]
> Please add some documentation about usefull helper function in tests

done

**Review by jeabakker — APPROVED** [review]

**Pull request merged**